src/realmFacade.js

```diff
@@ -92,10 +92,12 @@ export function buildChildRealm(unsafeRec, BaseRealm) {
       return r;
     }
 
-    static makeCompartment() {
+    static makeCompartment(options) {
+      options = Object(options); // todo: sanitize
```

Comment:
Still needs fixing, along with the old uses of this pattern prior to this PR.

Reply:
Done.